extensions/ql-vscode/src/vscode-tests/no-workspace/remote-queries/variant-analysis-results-manager.test.ts

```diff
@@ -33,8 +32,18 @@ describe(VariantAnalysisResultsManager.name, function() {
     variantAnalysisId = faker.datatype.number();
 
     try {
-      const extension = await extensions.getExtension<CodeQLExtensionInterface | Record<string, never>>('GitHub.vscode-codeql')!.activate();
-      cli = extension.cliServer;
+      cli = createMockCliServer({
+        bqrsInfo: [{ 'result-sets': [{ name: 'result-set-1' }, { name: 'result-set-2' }] }],
+        bqrsDecode: [{
+          columns: [{ kind: 'NotString' }, { kind: 'String' }],
+          tuples: [['a', 'b'], ['c', 'd']],
+          next: 1
+        }, {
+          columns: [{ kind: 'String' }, { kind: 'NotString' }, { kind: 'StillNotString' }],
+          tuples: [['a', 'b', 'c']]
+        }]
+      }, sandbox);
+      storagePath = path.join(__dirname);
```

Comment:
The `path.join` doesn't really seem to do anything here. Can we just use `__dirname` or even an empty storage path if it's not going to be used anyway?

Reply:
I've realized I forgot to move the dummy zip file we use for these tests into the `no-workspace` folder. I've now moved that file into `no-workspace/data/remote-queries` and set this dummy storage path to point to that folder for consistency: https://github.com/github/vscode-codeql/pull/1628/commits/da011c12b311811e9e3fb17a9b875749bfffc7e7 & https://github.com/github/vscode-codeql/pull/1628#:~:text=Set%20storagePath%20to%20point%20at%20data/remote%2Dqueries